integration-templates/hubspot/nango.yaml

```diff
@@ -0,0 +1,23 @@
+integrations:
+  hubspot:
+    hubspot-service-ticket:
+      runs: every half hour
+      returns:
+        - HubspotServiceTicket
+
+models:
+  HubspotServiceTicket:
+    id: integer
+    properties:
+      content: string
+      createdate: date
+      hs_lastmodifieddate: date
+      hs_object_id: integer
+      hs_pipeline: integer
+      hs_pipeline_stage: integer
+      hs_ticket_category: string | null
+      hs_ticket_priority: string
+      subject: string
```

Comment:
I think it would make sense to make the property names a bit more generic here, so they could later on be standardized with other CRM systems if needed. In particular: 
- have everything at the top level (i.e. kill the `properties` object)
- remove the `hs_` prefixes

Reply:
Yeah - that makes sense - I thought I needed to replicate the API response. Will make those changes